### ISSUE TITLE
Unable to find published extension

### ISSUE THREAD

**Issue by YouZag** [messagebird/firestore-send-msg#29]
`firebase ext:install -i`

... select the extension and get the message ...

Error: Unable to find published extension 'firestore-messagebird-send-msg'. Run firebase ext:install -i to select from the list of all available published extensions.

**Comment by aodinok** [comment]
Hi @YouZag 

Thank you for reporting.
I quickly checked the problem and from what I see it seems to me that the problem is on Firebase side.
It lists MB extension as `firebase/firestore-messagebird-send-msg` where the correct id of the extension should be `messagebird/firestore-messagebird-send-msg`.

@karayu @i14h can you guys please check why our extension is listed under `firestore/`?

@YouZag in the meantime you can use command with full extension name to install it:

`firebase ext:install messagebird/firestore-messagebird-send-msg  --project='YOUR_PROJECT_ID'`

Thanks, Oleks

**Comment by i14h** [comment]
@YouZag @aodinok yes this is a bug on our side and we're working on it. In the meantime please install the extension directly just as Oleksander mentioned.

**Comment by YouZag** [comment]
@i14h @aodinok Thanks for the quick responses.  Unfortunately, the full extension is how I originally ran into this problem as it also returns unable to find.

`firebase ext:install messagebird/firestore-messagebird-send-msg --project='MY_PROJECT_ID'`

i  extensions: ensuring required API firebaseextensions.googleapis.com is enabled...
✔  extensions: required API firebaseextensions.googleapis.com is enabled

Error: Unable to find published extension 'messagebird/firestore-messagebird-send-msg'. Run firebase ext:install -i to select from the list of all available published extensions.

**Comment by aodinok** [comment]
`firebase ext:install messagebird/firestore-messagebird-send-msg --project=ID`

Hmm it works for me: 

```
i  extensions: ensuring required API firebaseextensions.googleapis.com is enabled...
✔  extensions: required API firebaseextensions.googleapis.com is enabled
i  extensions: information about 'messagebird/firestore-messagebird-send-msg@latest':
Name: Send Messages with MessageBird
Publisher: messagebird
Description: Sends messages and notifications using the MessageBird Conversations API.
License: Apache-2.0
Source code: https://github.com/messagebird/firestore-send-msg/tree/master

? Would you like to continue installing this extension? (Y/n) 

```
Can you try again maybe the issue was already fixed by Firebase.

**Comment by aodinok** [comment]
Hey @i14h,

Is there any update on this? I still see that MessageBird extension is listed under `firebase`:
![Screenshot 2021-09-24 at 10 21 06](https://user-images.githubusercontent.com/11699084/134634568-7044474c-e5b2-46d7-877c-03d99dd8680b.png)
